lib/webpack/lint.js.webpack.js

```diff
@@ -9,15 +9,16 @@ const lint = (workingDir, conventionalConfig, mode) => {
     }),
     {}
   )
-  if (conventionalConfig.browserUpdateNote) {
-    inputFiles = {...inputFiles, browserUpdate: ['@sitegeist/conventional/lib/misc/browserupdate.js']}
-  }
 
   return {
     mode: mode,
     devtool: (conventionalConfig.sourceMaps) && mode !== 'production' ? 'source-map' : false,
     entry: inputFiles,
     context: workingDir,
+    output: {
+      path: path.resolve(workingDir, 'Resources/Public/tempAssets/'),
```

Comment:
We can't guarantee that this path exists. Maybe use a temporary directory? Maybe there is a node function that does this?

Reply:
good idea, we use a temporary directory for that purpose now.